sechub-webui/src/main/resources/application-basic-auth-mocked.yml

```diff
@@ -6,3 +6,9 @@ webui:
     ## Mocked sechub user (necessary for credentialinjection, we have no defaults...)
     userid: "mocked-user"
     apitoken: "mocked-apitoken"
+
+spring:
+  security:
+    user:
+      name: sechub
```

Comment:
I would prefer name and password where it is clear on first sight what the purpose of it is. Does also minimimze problems with secret scanners etc.

e.g. "mock-user" or "test-user" etc.